sdk/anomalydetector/azure-ai-anomalydetector/src/test/java/com/azure/ai/anomalydetector/AnomalyDetectorClientTestBase.java

```diff
@@ -27,7 +27,7 @@
  * Base class for Anomaly Detector clients test.
  */
 public class AnomalyDetectorClientTestBase extends TestBase {
-    private static final String FAKE_API_KEY = "1234567890";
+    private static final String FAKE_API_KEY = "fakeKeyPlaceholder";
```

Comment:
I don't see where this placeholder is defined in the suppression file.

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/31685/files#diff-1f8998f727116b38cd548541444ba882dd3306d2f5574840f571da0f3d61ba34L8

Reply:
For any inline code/value with keyword `placeholder`, Credscan will skip the checking. We don't need to suppress it literally in CredScanSuppression.json in this case. By having the inline suppression, we guarantee we suppress the place we expect instead of scanning all occurrences.